app/controllers/resources_controller.rb

```diff
@@ -1,12 +1,24 @@
 # frozen_string_literal: true
 
 class ResourcesController < ApplicationController
+  def index
+    render json: all_resources, status: :ok
+  end
+
   def show
     resource
   end
 
   private
 
+  def all_resources
+    if params[:system]
```

Comment:
tossing this comment here so brian can suggest the correct syntax.

for this filtering option, can we name the parameter `filter[system]` to coincide with the jsonapi spec around filtering? see: http://jsonapi.org/format/#fetching-filtering